canastra.js

```diff
@@ -4,6 +4,12 @@ var totalTeamA = 0;
 var totalTeamB = 0;
 
 function toSum() {
+  if (!elementA.value || !elementB.value) {
+    alert ("Please add a value!");

```

Comment:
This message looks like it is in a different language used by the user interface.